ygen/codegen.go

```diff
@@ -87,6 +87,18 @@ type GoOpts struct {
 	// produced if the GenerateJSONSchema YANGCodeGenerator field is set to
 	// true.
 	SchemaVarName string
+	// GoyangImportPath specifies the path that should be used in the generated
+	// code for importing the goyang/pkg/yang package. If it is not specified
```

Comment:
Suggest omitting the last sentence, here and below, since this duplicates the information already in the flag default.